packages/content/lib/utils.js

```diff
@@ -10,6 +10,7 @@ const getDefaults = ({ dev = false } = {}) => ({
   fullTextSearchFields: ['title', 'description', 'slug', 'text'],
   nestedProperties: [],
   markdown: {
+    tocDepth: 3,
```

Comment:
For backward compatibility, I set default value of `tocDepth` as 3, so that `h2` and `h3` are included in table of contents.